mmv1/templates/terraform/examples/vertex_ai_deployment_resource_pool.tf.erb

```diff
@@ -0,0 +1,19 @@
+resource "google_vertex_ai_deployment_resource_pool" "<%= ctx[:primary_resource_id] %>" {
+    region = "us-central1"
+    deployment_resource_pool_id = "sample-deployment-resource-pool"
```

Comment:
We'll want to define `vars` and supply the name through that- that will ensure that we add a random suffix and a sweeper prefix to test resources so that we don't leak resources or fail tests due to leaked resources.

Docs: https://googlecloudplatform.github.io/magic-modules/develop/test/#add-a-create-test

Reply:
I updated the code to use the random suffix through `vars` in [99bb911](https://github.com/GoogleCloudPlatform/magic-modules/pull/9518/commits/99bb911307b70377267cd15ebd7d230edabfdfee).